modules/phase_field/tests/GrandPotentialPFM/GrandPotentialMultiphase.i

```diff
@@ -244,21 +244,10 @@
     derivative_order = 2
     enable_jit = false
   [../]
-  [./FreeEng]
-    type = DerivativeParsedMaterial
-    args = 'etaa0 etab0 etab1 w'
-    f_name = F
-    constant_names =        'gbb  gab  m'
-    constant_expressions =  '1.5  4.5  1.0'
-    function = '1/4+m*(1/4*etaa0^4-1/2*etaa0^2+1/4*etab0^4-1/2*etab0^2+1/4*etab1^4-1/2*etab1^2
```

Comment:
Tagging @friedmud for an example where we used a parsed material to prototype a capability that is now implemented as a dedicated MOOSE class (Kernel). :-P